### PR TITLE
TAS-1310: iOS + Android issue on URL of job in job details (TestFlight)

### PR DESCRIPTION
**Pull request by bastantebom**
URL in copy job URL is showing capacitor as domain.

### CODE REVIEW

File: src/Nowruz/modules/Jobs/components/jobDetailAbout/index.tsx [hunk 1]
```diff
@@ -41,7 +43,12 @@ export const JobDetailAbout: React.FC<JobDetailAboutProps> = ({ isUser = true, s
   const [openApply, setOpenApply] = useState(false);
   const [openExternalApply, setOpenExternalApply] = useState(false);
   const [openAlert, setOpenAlert] = useState(false);
-  const url = window.location.href;
+  let url = window.location.href;
+  const extractedString = url.slice(url.indexOf('jobs'));
+  url =
+    isTouchDevice() && config.env === 'development'
+      ? `https://webapp2.dev.socious.io/${extractedString}`
```
Comment (jeyem) [inline src/Nowruz/modules/Jobs/components/jobDetailAbout/index.tsx:50]: no this code is not acceptable solution we don't use urls beyond our config , what is the problem to resolve it like this ?

**Review by jeyem — CHANGES_REQUESTED** [review]

File: src/Nowruz/modules/Jobs/components/jobDetailAbout/index.tsx (hunk 1 above)
Comment (bastantebom) [reply src/Nowruz/modules/Jobs/components/jobDetailAbout/index.tsx:50]: @jeyem this `window.location.href` would return `capacitor://capacitor.native` url. which wouldn't make more sense.

File: src/Nowruz/modules/Jobs/components/jobDetailAbout/index.tsx [hunk 2]
```diff
@@ -41,7 +43,12 @@ export const JobDetailAbout: React.FC<JobDetailAboutProps> = ({ isUser = true, s
   const [openApply, setOpenApply] = useState(false);
   const [openExternalApply, setOpenExternalApply] = useState(false);
   const [openAlert, setOpenAlert] = useState(false);
-  const url = window.location.href;
+  let url = window.location.href;
```
Comment (jeyem) [inline src/Nowruz/modules/Jobs/components/jobDetailAbout/index.tsx:46]:
```suggestion
  let url = `${config.appBaseURL}/jobs/${job.id}`
```

File: src/Nowruz/modules/Jobs/components/jobDetailAbout/index.tsx [hunk 3]
```diff
@@ -41,7 +43,12 @@ export const JobDetailAbout: React.FC<JobDetailAboutProps> = ({ isUser = true, s
   const [openApply, setOpenApply] = useState(false);
   const [openExternalApply, setOpenExternalApply] = useState(false);
   const [openAlert, setOpenAlert] = useState(false);
-  const url = window.location.href;
+  let url = `${config.appBaseURL}/jobs/${job.id}`
+  const extractedString = url.slice(url.indexOf('jobs'));
+  url =
```
Comment (jeyem) [inline src/Nowruz/modules/Jobs/components/jobDetailAbout/index.tsx:48]: need to be removed

File: src/Nowruz/modules/Jobs/components/jobDetailAbout/index.tsx [hunk 4]
```diff
@@ -41,7 +43,12 @@ export const JobDetailAbout: React.FC<JobDetailAboutProps> = ({ isUser = true, s
   const [openApply, setOpenApply] = useState(false);
   const [openExternalApply, setOpenExternalApply] = useState(false);
   const [openAlert, setOpenAlert] = useState(false);
-  const url = window.location.href;
+  let url = `${config.appBaseURL}/jobs/${job.id}`
+  const extractedString = url.slice(url.indexOf('jobs'));
```
Comment (jeyem) [inline src/Nowruz/modules/Jobs/components/jobDetailAbout/index.tsx:47]: need to be removed

File: src/Nowruz/modules/Jobs/components/jobDetailAbout/index.tsx [hunk 5]
```diff
@@ -41,7 +43,12 @@ export const JobDetailAbout: React.FC<JobDetailAboutProps> = ({ isUser = true, s
   const [openApply, setOpenApply] = useState(false);
   const [openExternalApply, setOpenExternalApply] = useState(false);
   const [openAlert, setOpenAlert] = useState(false);
-  const url = window.location.href;
+  let url = `${config.appBaseURL}/jobs/${job.id}`
```
Comment (jeyem) [inline src/Nowruz/modules/Jobs/components/jobDetailAbout/index.tsx:46]:
```suggestion
  let url = `${config.appBaseURL}/jobs/${jobDetail.id}`
```

**Review by jeyem — CHANGES_REQUESTED** [review]

File: src/Nowruz/modules/Jobs/components/jobDetailAbout/index.tsx (hunk 4 above)
Comment (bastantebom) [inline src/Nowruz/modules/Jobs/components/jobDetailAbout/index.tsx:47]:
```suggestion
  const extractedString = url.slice(url.indexOf('jobs'));
```

File: src/Nowruz/modules/Jobs/components/jobDetailAbout/index.tsx (hunk 3 above)
Comment (bastantebom) [inline src/Nowruz/modules/Jobs/components/jobDetailAbout/index.tsx:48]:
```suggestion
  url =
  
```

File: src/Nowruz/modules/Jobs/components/jobDetailAbout/index.tsx (hunk 4 above)
Comment (bastantebom) [inline src/Nowruz/modules/Jobs/components/jobDetailAbout/index.tsx]:
```suggestion
```

**Review by jeyem — APPROVED** [review]

**Review by Odeyaloleg — APPROVED** [review]

**Review by satanmourner — APPROVED** [review]

**Comment by satanmourner** [comment]
@bastantebom I approved your pull request and everything is fine, but you have one file change but 6 commits. consider when this PR is merged into the Main branch, all commits will be merged into the Main and these commits are not useful. (unless we use Squash merging)
it's better always to rebase your commits into one commit in this situation. (for better tacking history)
it's just a friendly comment and nothing else :> Thanks

**Pull request merged**